forc/src/ops/forc_init.rs

```diff
@@ -49,6 +49,12 @@ struct ContentResponse {
 
 pub fn init(command: InitCommand) -> Result<()> {
     let project_name = command.project_name;
+    if project_name == "test" {
+        bail!(
+            "the name `test` cannot be used as a package name, \
```

Comment:
I wonder if we should use `project name` instead of `package name` here and in `forc build`. When I write `forc init --help`, I get:
```
forc-init
Create a new Forc project

USAGE:
    forc init [OPTIONS] <PROJECT_NAME>

ARGS:
    <PROJECT_NAME>    The name of your project
```

Reply:
Also, we should probably check if case is important. Can we have `Test` as a project name?

Reply:
good catch, I'll update. Also, @adlerjohn if case is important here?

Reply:
For the sake of windows users, we should disallow `Test`/`TEST`/all case variants IMO.